### PR TITLE
Add orcid style to icons in record display

### PR DESCRIPTION
**Pull request by hoseinmirian**
followed and pasted the exact compact version of showing orcid in record display:
image

please test:
http://localhost:8080/FAIRsharing.2abjs5

### CODE REVIEW

**Review by knirirr — CHANGES_REQUESTED** [review]
The record page was already acceptable under ORCID's requirements, i.e. we had put an ORCID logo (with link) after the username.
Where our application needs changing is on the user pages, where the ID is shown as a non-hyperlinked ID with no logo (the logo should be before the ID).

**Comment by hoseinmirian** [comment]
@knirirr  this page you are referring to right?
http://localhost:8080/users/1136

**Comment by knirirr** [comment]
> @knirirr this page you are referring to right? http://localhost:8080/users/1136

Yes, that's the only page that needs changing to conform to ORCID's styles.

**Comment by hoseinmirian** [comment]
image

File: src/views/Users/PublicProfile.vue [hunk 1]
```diff
@@ -70,15 +70,15 @@
...
                       <v-list-item-content
-                        v-if="fieldName !== 'preferences'"
+                        v-if="fieldName !== 'preferences' && fieldName!=='orcid'"
                         class="py-0 d-block"
                       >
                         <b class="blue--text">{{ fieldName | cleanString }}: </b>
-                        <span v-if="field"> {{ field }} </span>
-                        <span v-else> None </span>
+                        <span v-if="field && fieldName!=='orcid'"> {{ field }} </span>
+                        <span> None </span>
```
Comment (knirirr) [inline src/views/Users/PublicProfile.vue]:
```suggestion
                        <span v-else> None </span>
```

**Review by knirirr — CHANGES_REQUESTED** [review]
Various spurious "None" entries are now appearing next to data fields:

https://deploy-preview-1360--fairsharing.netlify.app/#/users/1136

I think my suggested change will fix it.

**Review by knirirr — APPROVED** [review]

**Pull request merged**